src/documents/index.py

```diff
@@ -362,9 +362,16 @@ def _get_query(self) -> tuple:
         )
         q = qp.parse(q_str)
 
-        corrected = self.searcher.correct_query(q, q_str)
-        if corrected.query != q:
-            corrected.query = corrected.string
```

Comment:
Someone tell me if I'm crazy, but I think this _doesnt do anything_. It even looks incorrect setting the `query = string` (it's almost certainly a type mismatch).

I think the fix here is OK but arguably this could be just... removed?